server/database/migrations/000003_map.sql

```diff
@@ -1,12 +1,22 @@
 CREATE TYPE map.TILE_TYPE AS ENUM(
+       'EMPTY',
```

Comment:
Does it even make sense for a tile to be empty?

Reply:
You are bringing `Maybe not` to this conversation. For the sake of speed, I will say we use this for the 2D-array-using-1D-array arithmetic, so this needs to stay.